### PR TITLE
Add the annotation resources label to the SearchHit component.

### PR DESCRIPTION
**Pull request by jkeck**
Part of #2683

### CODE REVIEW

File: src/containers/SearchHit.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ const mapStateToProps = (state, { hit, companionWindowId, windowId }) => {
   const hitAnnotation = resourceAnnotations.find(r => r.id === hit.annotations[0]);
 
   return {
+    annotationLabel: hitAnnotation && hitAnnotation.resource && hitAnnotation.resource.label,
```
Comment (jkeck) [inline src/containers/SearchHit.js]: I don't think we can just use this `label` directly.  Just wanted to push this up, and happy to 🍐 up on how we should handle this properly (which maybe means pushing some of this to selectors)

File: src/state/selectors/searches.js [hunk 2]
```diff
@@ -83,3 +85,25 @@ export const getSelectedContentSearchAnnotations = createSelector(
     ),
   })).filter(val => val.resources.length > 0),
 );
+
+export const getResourceAnnotationForSearchHit = createSelector(
+  [
+    getSearchAnnotationForCompanionWindow,
+    (state, { hitId }) => hitId,
```
Comment (cbeer) [inline src/state/selectors/searches.js]: Is `hitId` the right name? Isn't this an annotation uri?

Comment (jkeck) [reply src/state/selectors/searches.js]: It's the it of the annotation in the hit (from https://github.com/ProjectMirador/mirador/blob/0a375b6e4bf5bc4c91976785fd7337ab21f88a05/src/containers/SearchHit.js#L26).  My intention was to differentiate it from the id in the resource, but agree that `annotationUri` may be less ambiguous.

Comment (cbeer) [reply src/state/selectors/searches.js]: `hitAnnotationUriThisIsTheOne`? :trollface:

**Pull request merged**